crates/rnote-ui/src/app/mod.rs

```diff
@@ -128,8 +128,15 @@ mod imp {
         /// Initializes and shows a new app window
         pub(crate) fn new_appwindow_init_show(&self, input_file: Option<gio::File>) {
             let appwindow = RnAppWindow::new(self.obj().upcast_ref::<gtk4::Application>());
+
             appwindow.init();
             appwindow.present();
+            // restore the window dimension and maximize after presenting it on mac os
+            // see issue 823 - https://github.com/flxzt/rnote/issues/823
+            #[cfg(target_os = "macos")]
+            if let Err(e) = appwindow.load_window_settings() {
+                tracing::error!("Failed to restore windows settings, Err: {e:}");
```

Comment:
The message should be `...{e:?}` so that the full error is printed verbosely

Reply:
I copied more or less what was in `crates/rnote-ui/src/appwindow/mod.rs` so there are a couple of `...{e}` that could be changed to `...{e:?}` there, should I go and change everything to `...{e:?}` there ?

Reply:
yes that would be nice!